### ISSUE TITLE
Tests broken on Node 22

### ISSUE THREAD

**Issue by fregante** [refined-github/refined-github#7425]
```
TypeError: Cannot set property navigator of #<Object> which has only a getter
 ❯ test/setup-file.js:15:23
     13| 
     14| for (const property of globals) {
     15|  globalThis[property] = window[property];
       |                       ^
     16| }
```

**Comment by fregante** [comment]
Due to https://github.com/nodejs/node/issues/39540#issuecomment-1770612761

**Comment by fregante** [comment]
I fixed that issue in #7470, but it looks like webpack can't find the config file on Windows 🤷‍♂️ 

```
> x:webpack
> cross-env NODE_OPTIONS="--import tsx --no-warnings=ExperimentalWarning" webpack --mode=production webpack-stats.json

[webpack-cli] Invalid configuration object. Webpack has been initialized using a configuration object that does not match the API schema.
 - configuration.entry[0] should be a non-empty string.
   -> A module that is loaded upon startup. Only the last one is exported.
```
